style.css

```diff
@@ -0,0 +1,3 @@
+.col {
+  color: rgb(0 255 255 / 50%);
+}
```

Comment:
- [ ] Great job creating a `style.css` file ✌🏼, but if you check your website the color is not taking effect it is because you forgot to add the reference into the .HTML file. Kindly, add the following line of code inside the `<head>` tag in your `index.html` file 🪜.

~~~ bash
<link rel="stylesheet" href="style.css">
~~~